components/api/api-modules-javascript/src/main/resources/META-INF/dirigible/modules/src/http/request.ts

```diff
@@ -1,232 +1,233 @@
-};
-
-export function getAttribute(name) {
-    return HttpRequestFacade.getAttribute(name);
-};
-
-export function getAuthType() {
-    return HttpRequestFacade.getAuthType();
-};
-
-export function getCookies() {
-    let cookiesJson = HttpRequestFacade.getCookies();
-    return JSON.parse(cookiesJson);
-};
-
-export function getAttributeNames() {
-    let attrNamesJson = HttpRequestFacade.getAttributeNames();
-    return JSON.parse(attrNamesJson);
-};
-
-export function getCharacterEncoding() {
-    return HttpRequestFacade.getCharacterEncoding();
-};
-
-export function getContentLength() {
-    return HttpRequestFacade.getContentLength();
-};
-
-export function getHeaders(name) {
-    let headersJson = HttpRequestFacade.getHeaders(name);
-    return JSON.parse(headersJson);
-};
-
-export function getContentType() {
-    return HttpRequestFacade.getContentType();
-};
-
-export function getBytes() {
-    let bytesJson = HttpRequestFacade.getBytes();
-    return JSON.parse(bytesJson);
-};
-
-let textData = null;
-export function getText() {
-    let textData = null;
-    if (textData === null) {
-        textData = HttpRequestFacade.getText();
-    }
-    return textData;
-};
-
-export function json() {
-    return getJSON();
-}
+export class Request {
+    public static isValid(): boolean {
+        return HttpRequestFacade.isValid();
+    };
+
+    public static getMethod(): string {
+        return HttpRequestFacade.getMethod();
+    };
+
+    public static getRemoteUser(): string {
+        return HttpRequestFacade.getRemoteUser();
+    };
+
+    public static getPathInfo(): string {
+        return HttpRequestFacade.getPathInfo();
+    };
+
+    public static getPathTranslated(): string {
+        return HttpRequestFacade.getPathTranslated();
+    };
+
+    public static getHeader(name: string): string {
+        return HttpRequestFacade.getHeader(name);
+    };
+
+    public static isUserInRole(role: string): boolean {
+        return HttpRequestFacade.isUserInRole(role);
+    };
+
+    public static getAttribute(name: string): string {
+        return HttpRequestFacade.getAttribute(name);
+    };
+
+    public static getAuthType(): string {
+        return HttpRequestFacade.getAuthType();
+    };
+
+    public static getCookies(): string {
+        let cookiesJson = HttpRequestFacade.getCookies();
+        return JSON.parse(cookiesJson);
+    };
+
+    public static getAttributeNames(): string {
+        let attrNamesJson = HttpRequestFacade.getAttributeNames();
+        return JSON.parse(attrNamesJson);
+    };
+
+    public static getCharacterEncoding(): string {
+        return HttpRequestFacade.getCharacterEncoding();
+    };
+
+    public static getContentLength(): number {
+        return HttpRequestFacade.getContentLength();
+    };
+
+    public static getHeaders(name: string): string {
+        let headersJson = HttpRequestFacade.getHeaders(name);
+        return JSON.parse(headersJson);
+    };
```

Comment:
I think here the return type is not `string`, it's either `Object`, `Object[]`, or something more tailored.